apps/web/src/components/tx/DecodedTx/index.tsx

```diff
@@ -1,15 +1,77 @@
-import { type SyntheticEvent, type ReactElement, memo } from 'react'
+import { type SyntheticEvent, type ReactElement, memo, useMemo } from 'react'
 import { isCustomTxInfo, isNativeTokenTransfer, isTransferTxInfo } from '@/utils/transaction-guards'
-import { Accordion, AccordionDetails, AccordionSummary, Box, Stack } from '@mui/material'
+import {
+  Accordion,
+  accordionClasses,
+  AccordionDetails,
+  AccordionSummary,
+  accordionSummaryClasses,
+  Box,
+  Stack,
+  styled,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import type { Palette } from '@mui/material'
 import { type SafeTransaction } from '@safe-global/safe-core-sdk-types'
-import type { TransactionDetails } from '@safe-global/safe-gateway-typescript-sdk'
+import { TransactionInfoType, type TransactionDetails } from '@safe-global/safe-gateway-typescript-sdk'
 import Summary from '@/components/transactions/TxDetails/Summary'
 import { trackEvent, MODALS_EVENTS } from '@/services/analytics'
 import Multisend from '@/components/transactions/TxDetails/TxData/DecodedData/Multisend'
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 import DecodedData from '@/components/transactions/TxDetails/TxData/DecodedData'
 import accordionCss from '@/styles/accordion.module.css'
 import HelpToolTip from './HelpTooltip'
+import { useDarkMode } from '@/hooks/useDarkMode'
+
+enum ColorLevel {
+  info = 'info',
+  warning = 'warning',
+  success = 'success',
+}
+
+const TX_INFO_LEVEL = {
+  [ColorLevel.warning]: [TransactionInfoType.SETTINGS_CHANGE],
```

Comment:
Does this cover fallback handler/modules/guards like your previous implementation?